### DISCUSSION TITLE
Friendly SMTP

### DISCUSSION THREAD

**Discussion by revolunet** [betagouv/cap#21] [Autre]
Quel fournisseur ~~éthique~~ fiable pour les envois transactionnels et newsletters ?

Il nous faut une bonne déliberavilité et pas de tracking pour les usagers

**Comment by adipasquale** [comment]
L’état des lieux c’est que Brevo est un très bon fournisseur français qui coche plein de cases mais sur la question du tracking dans les mails laisse un peu à désirer. Ils fournissent (entre autres) 2 fonctionnalités de tracking : 

- l’inclusion de pixels invisibles pour tracker les ouvertures
- la réécriture des URLs dans les emails pour passer par leurs serveurs et pouvoir tracker les clics

Ces 2 fonctionnalités sont activées par défaut, sans vraiment de visibilité pour les non avertis.
Le plus problématique est qu’on ne peut pas désactiver ces fonctionnalités simplement, voire du tout.

On essaie de négocier avec eux dans le support pour une SE mais il faut se justifier et on ne sait pas encore si on y arrivera

**Reply by sneko** [reply]
@jdauphant il me semble que tu as suivi en partie ce sujet. N'hésite pas à y apporter des informations 👍

**Reply by jdauphant** [reply]
Vous avez demandé un avis d'un juriste sur domain-juridique ?
Vous aurez des points pour appuyer votre demande ou le remettre en question s'il n'y a pas de fondement juridique.

**Reply by thbar** [reply]
> Le plus problématique est qu’on ne peut pas désactiver ces fonctionnalités simplement, voire du tout.

Mon impression personnelle est que cette fonction est assez profondément "engrainée" dans leur produit et que ça va être compliqué.

Pour moi c'est un point bloquant surtout quand on voit ce que ça donne sur une invitation email à un sujet sensible (gestionnaire de mot de passe) - on se retrouve avec des mails en `gjbbdae.r.bh.d.sendibt3.com` typique d'un phishing, on va s'y habituer, et après on aura droit à un vrai phishing, et la personne se fera avoir.

**Reply by adipasquale** [reply]
cf https://mattermost.incubateur.net/betagouv/pl/9tnu176m93bpdpm5p66ad73xbc

**Comment by adipasquale** [comment]
en alternatives potentielles pour les mails transactionnels : 

- https://fr.tipimail.com/ (ex sarbacane) est une entreprise française. Ils nous ont dit permettre de désactiver ces fonctionnalités de tracking dans leur support, je n’ai pas testé

**Reply by revolunet** [reply]
j'avais testé et bon service, SMTP et API dispos, gestion des sous-comptes possible

**Reply by tut-tuuut** [reply]
Chez Aidants Connect on avait aussi Tipimail, c'était pas mal oui !
Et en 2022 ils étaient dispo à l'UGAP.

**Comment by johanricher** [comment]
Autre alternative : https://www.scaleway.com/fr/email-transactionnel-tem/

**Comment by johanricher** [comment]
> fournisseur éthique

Sur quels critères objectifs ou définition on juge ça ?

**Reply by revolunet** [reply]
oui pas clair, j'ai corrigé :) j'entendais par là un fournisseur qui a une solution pour l'envoi sans tracking et qui "comprend" ce sujet.

**Comment by revolunet** [comment]
on me namedrop https://mailpace.com/ (UK)

**Comment by jdauphant** [comment]
Brevo a désactivé le tracking pour rdv-solidarité.
On prépare les arguments juridiques et on va en parler de manière posé avec Brevo dans le cadre du suivi de notre contrat.

**Comment by jdauphant** [comment]
Nous avons vu Brevo hier.

Brevo désactivera le tracking (ouverture et clic sur des liens) de tous les comptes rattachés à notre compte entreprise qui le demande au travers de notre conseiller entreprise (vous pouvez désactiver ouverture ou/et clic). 

Pour cela, il faut avoir une IP dédié (ils utilisent le tracking pour surveiller la réputation des IPs et ne prennent pas de risque pour les IPs partagés entre plusieurs clients).
Il est surement possible en s'appuyant sur ces 2 éléments de faire la demande sur des comptes à part. 

Il faut savoir que vous perdez bien évidemment les statistiques d'envoi (ouverture et clic sur les liens) et vous ne pourrez pas savoir si vous avez des problèmes d'envois et vers quel fournisseur (par exemple Orange bloque votre IP), dans ce cas Brevo ne pourra pas vous aider.

Brevo propose en alternative de supprimer automatiquement toutes les données collectés (l'option est dans les paramètres).

Il faut contacter le charger de compte Brevo pour faire retirer les options.

**Reply by thbar** [reply]
> vous ne pourrez pas savoir si vous avez des problèmes d'envois et vers quel fournisseur (par exemple Orange bloque votre IP), dans ce cas Brevo ne pourra pas vous aider.

Ah c'est tout ou rien ? Il serait souhaitable de dissocier le tracking sur les liens, du tracking des "bounce" qui ont une utilité différente... Merci pour l'update par contre !!

**Comment by jdauphant** [comment]
@revolunet on peut supprime le tracking de clic sur les envois des apps animations (vaultwarden, mattermost ...)

**Reply by revolunet** [reply]
yes please

**Comment by revolunet** [comment]
a suivre: https://beta.gouv.fr/startups/notifications-v2.html

**Comment by goulvench** [comment]
Et puisque c'est bientôt Noël, on peut demander un SMTP entrant aussi ? 

Collectif Objets utilise l'API de Brevo pour recevoir des mails, mais Rails (via [ActionMailBox](https://guides.rubyonrails.org/action_mailbox_basics.html)) permet de s'interfacer directement avec Exim, Postmark, etc, pour ne pas dépendre d'un fournisseur spécifique, et ça serait top.

**Comment by revolunet** [comment]
related article: Pourquoi nous migrons de Mailgun vers Scaleway pour vous offrir une infrastructure e-mail 100% souveraine

https://bearstech.com/societe/blog/pourquoi-nous-migrons-de-mailgun-vers-scaleway-pour-vous-offrir-une-infrastructure-e-mail-100
